test/integration/snapshot_test.go

```diff
@@ -21,18 +21,42 @@ package integration
 import (
 	"context"
 	"fmt"
-	"github.com/yorkie-team/yorkie/pkg/document/json"
+	"log"
+	"math"
+	"reflect"
 	"testing"
-	"time"
+
+	"bou.ke/monkey"
+	"github.com/yorkie-team/yorkie/pkg/document/json"
 
 	"github.com/stretchr/testify/assert"
 
+	"github.com/yorkie-team/yorkie/api/types"
+	"github.com/yorkie-team/yorkie/client"
 	"github.com/yorkie-team/yorkie/pkg/document"
 	"github.com/yorkie-team/yorkie/pkg/document/key"
+	"github.com/yorkie-team/yorkie/server"
+	"github.com/yorkie-team/yorkie/server/backend/background"
+	"github.com/yorkie-team/yorkie/server/backend/database/mongo"
+	"github.com/yorkie-team/yorkie/server/logging"
 	"github.com/yorkie-team/yorkie/test/helper"
 )
 
 func TestSnapshot(t *testing.T) {
+	var b *background.Background
+	// "bou.ke/monkey"
+	monkey.PatchInstanceMethod(
+		reflect.TypeOf(b),
+		"AttachGoroutine",
+		func(_ *background.Background, f func(c context.Context)) {
+			f(context.Background())
+		},
+	)
+	defer monkey.UnpatchInstanceMethod(
+		reflect.TypeOf(b),
+		"AttachGoroutine",
+	)
```

Comment:
How about using unpatch directly?

```suggestion
	guard := monkey.PatchInstanceMethod(
		reflect.TypeOf(b),
		"AttachGoroutine",
		func(_ *background.Background, f func(c context.Context)) {
			f(context.Background())
		},
	)
	defer func() {
		guard.Unpatch()
	}()
```

Reply:
Oh, I see. Thanks for the explanation.